src/mark.ts

```diff
@@ -81,22 +81,22 @@ export interface MarkConfig extends VgMarkConfig {
   color?: string;
 
   /**
-   * Offset for x channel in pixels. The offset is applied before rotation by the _angle_ property.
+   * Offset for x channel in pixels.
```

Comment:
actually, one more nitpick --  
Can we say 
 "Offset for x-position"

instead of x channel